docs/client-specifications/error-handling.md

```diff
@@ -71,7 +71,7 @@ enum MomentoErrorCode {
   PERMISSION_ERROR = 'PERMISSION_ERROR',                   // message: `Insufficient permissions to perform an operation on a cache: ${detail}`
   AUTHENTICATION_ERROR = 'AUTHENTICATION_ERROR',           // message: `Invalid authentication credentials to connect to cache service: ${detail}`
   CANCELLED_ERROR = 'CANCELLED_ERROR',                     // message: `The request was cancelled by the server; please contact Momento: ${detail}`
-  LIMIT_EXCEEDED_ERROR = 'LIMIT_EXCEEDED_ERROR',           // message: `Request rate exceeded the limits for this account.  To resolve this error, reduce your request rate, or contact Momento to request a limit increase.`
+  LIMIT_EXCEEDED_ERROR = 'LIMIT_EXCEEDED_ERROR',           // message: `Request rate, bandwidth, or object size exceeded the limits for this account.  To resolve this error, reduce your usage as appropriate or contact Momento to request a limit increase.`
```

Comment:
Just thinking out loud and I'm good merging as is but might be better to break out type error to better describe type of limit. Just feel it's sometimes hard for users to figure out which they hit.